src/http_server.cpp

```diff
@@ -315,6 +322,10 @@ namespace sim
 		m_recv_buffer.erase(m_recv_buffer.begin(), m_recv_buffer.begin() + req_len);
 		m_bytes_used -= req_len;
 
+		if (m_send_buffer.empty()) {
```

Comment:
right now if a handler isn't found in ``m_handlers``, we return a 404. there could be a separate map of paths that should just stall the connection. If we don't find a normal handler, then look up in there to see if we should return 404 or stall. that way the existing handlers aren't overloaded with new behavior.